source/main/datatypes/wheel_t.h

```diff
@@ -24,50 +24,53 @@
 #include "RoRPrerequisites.h"
 
 /**
-* SIM-CORE; Wheel.
-*/
+ * SIM-CORE; Wheel.
+ */
 struct wheel_t
 {
-    enum class BrakeCombo /// Wheels are braked by three mechanisms: A footbrake, a handbrake/parkingbrake, and directional brakes used for skidsteer steering.
+    enum class BrakeCombo /// Wheels are braked by three mechanisms: A footbrake, a handbrake/parkingbrake, and directional brakes
```

Comment:
I'd go with ReflowComments:False - I don't think it's important to have the comments visible in whole. The developer will read the comment once to grok the code and then it'll just distract him. I find DearIMGUI's comment-management inspirational: https://github.com/ocornut/imgui/blob/master/imgui.h#L264